### ISSUE TITLE
Got Chrome 4 as earliest supported version

### ISSUE THREAD

**Issue by max-mykhailenko** [browserslist/browserslist#472]
Commands and results
`npx browserslist '> 0.5% in my stats'`

Result
```
and_chr 80
android 80
chrome 80
chrome 70
firefox 74
ios_saf 13.4
ios_saf 13.3
ios_saf 13.0-13.1
ios_saf 12.2-12.4
safari 13
```
Looks like it's exactly what I have in my `browserslist-stats.json`.

But when I use `babel/preset-env` and pass the same string/same config got next (debug mode on)

```
Using targets:
{
  "android": "80",
  "chrome": "4",
  "firefox": "74",
  "ios": "12.2",
  "safari": "13"
}
```

When I pass all browsers' version into babel-env directly I don't have problems with build.

Versions
`"@babel/core": "7.9.0"`

**Comment by ai** [comment]
Can you show results of `npx browserslist …` and `npx broeserslist --mobile-to-desktop`.

It will allow us to be sure, that it is not Babel problem.

**Comment by max-mykhailenko** [comment]
@ai looks like `"@babel/core": "7.9.0"` installs `browserslist` version which doesn't have `--mobile-to-desktop` arg

**Comment by max-mykhailenko** [comment]
```npm info browserslist version
4.11.1
```

**Comment by max-mykhailenko** [comment]
`Babel-env` installs `browserslist` version 4.10.0

**Comment by max-mykhailenko** [comment]
```
npx browserslist --mobile-to-desktop '> 0.5% in my stats'
browserslist: Unknown arguments --mobile-to-desktop.

Usage:
  npx browserslist
  npx browserslist "QUERIES"
  npx browserslist --json "QUERIES"
  npx browserslist --config="path/to/browserlist/file"
  npx browserslist --coverage "QUERIES"
  npx browserslist --coverage=US "QUERIES"
  npx browserslist --coverage=US,RU,global "QUERIES"
  npx browserslist --env="environment name defined in config"
  npx browserslist --stats="path/to/browserlist/stats/file"
  npx browserslist --update-db
```

**Comment by ai** [comment]
Start the new project without any dependency expect `browserslist` to avoid the problem of having old dependencies in `node_modules`.

**Comment by ai** [comment]
You may use different versions of Browserslist in `npx` and Babel. Your problem may be already fixed in the latest version.

**Comment by max-mykhailenko** [comment]
@ai thanks for advise. I move only file `browserslist-stats.json`
```
npx browserslist --mobile-to-desktop '> 0.5% in my stats'
npx: installed 11 in 2.251s
and_chr 4
android 80
chrome 80
chrome 70
firefox 74
ios_saf 13.4
ios_saf 13.3
ios_saf 13.0-13.1
ios_saf 12.2-12.4
safari 13


npx browserslist '> 0.5% in my stats'
npx: installed 11 in 2.154s
and_chr 80
android 80
chrome 80
chrome 70
firefox 74
ios_saf 13.4
ios_saf 13.3
ios_saf 13.0-13.1
ios_saf 12.2-12.4
safari 13
```

**Comment by ai** [comment]
Thanks for the reproduction stand. I will not have a time next week, because of different open source projects.

Can I ask you to review this code in your test stand? https://github.com/browserslist/browserslist/blob/master/index.js#L248-L260

**Comment by max-mykhailenko** [comment]
I'll review during this week. May I ask your advise about condition like '> 0.5% in my stats but chrome > 70'. Is it possible somehow with the current codebase?

**Comment by ai** [comment]
> May I ask your advise about condition like '> 0.5% in my stats but chrome > 70'. Is it possible somehow with the current codebase?

```
> 0.5% in my stats, not chrome < 70
```

**Comment by aronwoost** [comment]
I'm seeing the same issue.

How can I assist to debug this issue? 

When doing `npx browserslist --mobile-to-desktop '> 0.5% in my stats'` I'm also seeing `and_chr 4` which is causing `"chrome": "4"`, right?

**Comment by aronwoost** [comment]
I digged into the code a bit.

The problem is [`normalizeStats()`](https://github.com/browserslist/browserslist/blob/3bfe8ca5188bd060fd7e395835aaa53d9dd7b08a/node.js#L115) together with `mobileToDesktop` being `true`.

Since both arguments in `normalizeStats(data, stats)` have one version for `and_chr`, the output is `{and_chr: { '0': 1 }}` (check [this condition](https://github.com/browserslist/browserslist/blob/3bfe8ca5188bd060fd7e395835aaa53d9dd7b08a/node.js#L125)). Is assume this is correct and by design (I'm not fully aware of the architecture).

Then, since `mobileToDesktop` is `true`, `chrome` is [cloned](https://github.com/browserslist/browserslist/blob/3bfe8ca5188bd060fd7e395835aaa53d9dd7b08a/index.js#L253) with all the chrome entries and `{and_chr: { '0': 1 }}` results in picking the first `chrome` version, which is 4.

I'm willing to work on this issue, but I'm unsure where to tackle it. I don't want to make `normalizeStats()` aware of `mobileToDesktop`. We could quick-fix it in `byName()` by adding another if. But I'm not sure about it.

**Comment by ai** [comment]
Can you send PR? It is hard to understand your fix in text description.

**Comment by aronwoost** [comment]
@ai True, absolutely impossible to understand! 😂😂😂

I'm willing to fix, but I'm unsure where. 

You were correct above 👆🏼, `byName()` does not work as expected.

I provided a failing test in #572. Does this help?

**Comment by ai** [comment]
@aronwoost can you check this PR? Maybe we already fixed the problem

https://github.com/browserslist/browserslist/pull/511

**Comment by aronwoost** [comment]
@ai 

Same result in the PR

```
    Expected substring: "and_chr 87"
    Received string:    "and_chr 4"
```

**Comment by Semigradsky** [comment]
@aronwoost your test can be fixed by changing
https://github.com/browserslist/browserslist/blob/3bfe8ca5188bd060fd7e395835aaa53d9dd7b08a/node.js#L130
to
```js
      var normal = data[i].versions[0]
```
Could you try this change?

@ai As I see `data[i].versions` is array of strings. Maybe earlier it could have been an object?

**Comment by aronwoost** [comment]
@Semigradsky The change does indeed fixes the test. But it does not fixes the `browserslist --mobile-to-desktop '> 0.5% in my stats'` issue.

Sorry for pointing in the wrong direction. 😕

Let me see if I can create a better test.

**Comment by ai** [comment]
The fix released in 4.16.1.
